webpages/settings/index.js

```diff
@@ -126,3 +126,9 @@ chrome.runtime.sendMessage("getSettingsInfo", ({ manifests, addonsEnabled, addon
 vue.$watch("selectedTab", function (newSelectedTab) {
   this.selectedTag = null;
 });
+window.addEventListener("keydown", function (e) {
+  if (e.keyCode === 114 || (e.ctrlKey && e.keyCode === 70)) {
+    e.preventDefault();
+    $("#searchBox").focus();
```

Comment:
I don't think there's jQuery in this context. This might have worked in the console because `$` is a shortcut for `document.querySelector`, but I don't think this would actually work.